src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -536,6 +520,41 @@ private Path createJNIConfig(String suffix) throws IOException {
         return jniPath;
     }
 
+    private Path createResourceConfig(String suffix) throws IOException {
+        Path gvmPath = paths.getGvmPath();
+        Path resourcePath = gvmPath.resolve(
+                Strings.substitute(Constants.RESOURCE_ARCH_FILE, Map.of("archOs", suffix)));
+        Files.deleteIfExists(resourcePath);
+        try (BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(new FileOutputStream(resourcePath.toFile())))) {
+            bw.write("{\n");
+            bw.write("  \"resources\": [\n");
+            boolean patternHasBeenWritten = false;
+            for (String extension : RESOURCES_BY_EXTENSION) {
```

Comment:
Maybe we could create a file with the default resources instead of writing it every time? Only adding the user list will be required.